var/spack/repos/builtin/packages/apex/package.py

```diff
@@ -97,13 +98,14 @@ class Apex(CMakePackage):
     variant("lmsensors", default=False, description="Enables LM-Sensors support")
     variant("mpi", default=False, description="Enables MPI support")
     variant("starpu", default=False, description="Enables StarPU support")
+    variant("opencl", default=False, description="Enables OpenCL support")
```

Comment:
Consider
```suggestion
    variant("opencl", default=False, description="Enables OpenCL support", when="@2.7:)
```
instead of the conflict? Not a blocker.

Reply:
Yeah, the support was added in 2.7, so this is a good suggestion - thanks. The conflict is because I foolishly didn't make the OpenCL support optional. If/when I get around to that, I'll make it so that SYCL and OpenCL can both be configured/built at the same time, and the user can choose the right support at run time.